### ISSUE TITLE
Link to API docs

### ISSUE THREAD

**Issue by kwalrath** [dart-lang/angular_components_example#75]
The docs in the gallery consist only of the class description. Often that's all you need, but sometimes you might want more API details. You shouldn't have to look at the source code to get those.

Also, links in the description come out plain. E.g. [SelectionOptions] in the [MaterialSelectComponent gallery page](https://dart-lang.github.io/angular_components_example/#/material_select) is plain text, but is a real link in the [MaterialSelectComponent API doc](https://pub.dartlang.org/documentation/angular_components/0.9.0-alpha+10/material_select_material_select/MaterialSelectComponent-class.html). That seems like a bug, but having easy access to the API doc would make it less of a problem.

**Comment by nshahan** [comment]
I agree there is room for improvement. I'm focusing my work this quarter to improve the documentation pages that are generated in the gallery. 

The missing link that you described isn't a bug but rather just a feature that hasn't been implemented because it isn't clear what it should actually link to since not all of the classes that you can reference in the dartdoc are represented in the gallery.

**Comment by chalin** [comment]
FYI, the AngularTS material app has a separate tab for the API entry ("Overview" and "Examples" being the other tabs), e.g. see https://material.angular.io/components/sidenav/api:

> <img width="1054" alt="screen shot 2018-05-01 at 11 41 10" src="https://user-images.githubusercontent.com/4140793/39479889-9920734e-4d34-11e8-8a20-799376696f40.png">

**Comment by nshahan** [comment]
These were recently updated to appear as code instead of the square brackets. I'll leave this as a possible improvement in the future for full dartdoc linking support.
